spec/controllers/concerns/rate_limit_concern_spec.rb

```diff
@@ -98,4 +98,69 @@ def update
       end
     end
   end
+
+  describe '#confirm_not_rate_limited_after_doc_auth' do
+    controller Idv::StepController do
+      before_action :confirm_not_rate_limited_after_doc_auth
+    end
+
+    before(:each) do
+      sign_in(user)
+      allow(subject).to receive(:current_user).and_return(user)
+      routes.draw do
+        get 'show' => 'idv/step#show'
+        put 'update' => 'idv/step#update'
+      end
+    end
+
+    it 'redirects if the user is rate limitted for a step after doc auth' do
```

Comment:
typo, several places
```suggestion
    it 'redirects if the user is rate limited for a step after doc auth' do
```